include/zephyr/net/wifi_nm.h

```diff
@@ -94,6 +94,10 @@ int wifi_nm_unregister_mgd_iface(struct wifi_nm_instance *nm, struct net_if *ifa
  * @}
  */
 
+#define SSID_MAX_LEN 32
+#define ETH_ALEN 6
+#define MAC_STR_FORMAT "%2hhx:%2hhx:%2hhx:%2hhx:%2hhx:%2hhx"
```

Comment:
Hmm, I think this does not belong to a header file so please just place it in the .c file where it is used.

Reply:
Added this to header so that applications can re-use this instead of defining it again in case where they may be printing MAC address. Let me know if you agree. Else, I can move it to .c file.

Reply:
Here the name of the define is not namespaced and I feel that normally these kind of format specifiers are not good to be placed in header file (unless inside inline function etc) because user of these need to be careful to have proper parameters in printf statements as one needs to see two files. And if the define in header file is changed, one needs to make sure that all the users of that are changed too. This can be problematic for the external modules/sources.
Perhaps this is not a real problem in this case but just to be in safe side and a general rule better to place these to .c file only.

Reply:
Understood. Will move it to .c file.

Reply:
Done.